compiler/damlc/tests/daml-test-files/Iou_template.EXPECTED.rst

```diff
@@ -12,7 +12,6 @@ Templates
 
```

Comment:
Can we get a test that actually uses the links in type signatures?

Reply:
I'll add one 👍